packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -385,13 +386,22 @@ export default function (
             if (dynamicOpts) dynamicOpts(task.options, task);
 
             const throwMsg = task.options.throws;
+            const rejectMsg = task.options.rejects;
             if (throwMsg) {
               // internal api doesn't have this option but it's best not to pollute
               // the options object with useless options
               delete task.options.throws;
 
-              assert.throws(runTask, function (err) {
-                return throwMsg === true || err.message.indexOf(throwMsg) >= 0;
+              return assert.throws(runTask, function (err) {
```

Comment:
Though `assert.rejects` shares same behaviour with `assert.throws` on synchronous errors, which means we can simply replace `assert.throws` by `assert.rejects` and extend `throws` so that it contains both sync and async errors. However I think it is still valuable to differentiate between them.